opencga-app/src/main/java/org/opencb/opencga/app/cli/main/options/OrganizationsCommandOptions.java

```diff
@@ -216,6 +218,47 @@ public class UpdateNotesCommandOptions {
     
     }
 
+    @Parameters(commandNames = {"configuration-update"}, commandDescription ="Update the Organization configuration attributes")
+    public class UpdateConfigurationCommandOptions {
+    
+        @ParametersDelegate
+        public CommonCommandOptions commonOptions = commonCommandOptions;
+    
+        @Parameter(names = {"--json-file"}, description = "File with the body data in JSON format. Note, that using this parameter will ignore all the other parameters.", required = false, arity = 1)
+        public String jsonFile;
+    
+        @Parameter(names = {"--json-data-model"}, description = "Show example of file structure for body data.", help = true, arity = 0)
+        public Boolean jsonDataModel = false;
+    
+        @Parameter(names = {"--include", "-I"}, description = "Fields included in the response, whole JSON path must be provided", required = false, arity = 1)
+        public String include; 
+    
+        @Parameter(names = {"--exclude", "-E"}, description = "Fields excluded in the response, whole JSON path must be provided", required = false, arity = 1)
+        public String exclude; 
+    
+        @Parameter(names = {"--organization"}, description = "Organization id", required = true, arity = 1)
+        public String organization; 
+    
+        @Parameter(names = {"--include-result"}, description = "Flag indicating to include the created or updated document result in the response", required = false, help = true, arity = 0)
+        public boolean includeResult = false; 
+    
+        @Parameter(names = {"--authentication-origins"}, description = "Action to be performed if the array of authenticationOrigins is being updated.", required = false, arity = 1)
+        public String authenticationOrigins = "ADD"; 
+    
+        @Parameter(names = {"--optimizations-simplify-permissions"}, description = "The body web service simplifyPermissions parameter", required = false, help = true, arity = 0)
+        public boolean optimizationsSimplifyPermissions = false;
+    
+        @Parameter(names = {"--token-algorithm"}, description = "The body web service algorithm parameter", required = false, arity = 1)
```

Comment:
Missing param descriptions

Reply:
That's automatically generated. I know we need to change those description values :S

Reply:
What I meant was that you should add these descriptions with "@DataField"  in the "Organization configuration"